frontend/src/app/projects/project-detail/create-project-tools/create-project-tools.component.html

```diff
@@ -0,0 +1,56 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="flex justify-center">
```

Comment:
From the project details page, we have three different subviews (Link Tool, Configure Project, and Configure Model). All three subviews have a different positions for the cards they contain (full-width, centered, on the left). It would be nice for these views to look more consistent

Reply:
Since it might affect other components as well and is more of a general consistency issue, I've transfered to a new issue: https://github.com/DSD-DBS/capella-collab-manager/issues/2023